templates/shared/_cube-contact-us-form.html

```diff
@@ -11,25 +11,105 @@ <h1>{{h1}}</h1>
         <fieldset class="u-no-margin--bottom">
           <h3>About you</h3>
           <ul class="p-list">
-            <li class="mktFormReq mktField p-list__item">
-              <label for="firstName" class="mktoLabel">First name:</label>
-              <input required id="firstName" name="firstName" maxlength="255" type="text" class="mktoField mktoRequired" />
+            <li  class="p-list__item  ">
+              <label for="FirstName" id="LblFirstName"  >First name:</label>
+              <input id="FirstName" name="firstName" maxlength="255" aria-labelledby="LblFirstName" type="text" class="is-required" required="" aria-required="true" >
             </li>
-            <li class="mktFormReq mktField p-list__item">
-              <label for="lastName" class="mktoLabel">Last name:</label>
-              <input required id="lastName" name="lastName" maxlength="255" type="text" class="mktoField mktoRequired" />
+
+            <li  class="p-list__item  ">
+              <label for="LastName" id="LblLastName"  >Last name:</label>
+              <input id="LastName" name="lastName" maxlength="255" aria-labelledby="LblLastName InstructLastName" type="text" class="is-required" required="" aria-required="true" >
```

Comment:
I thought these ids needed to be camel case instead of sentence case?

Reply:
I thought only the `name=` mattered?